### PR TITLE
[cu-33a8fw3] Refactor Pablo to handle updated events

### PR DESCRIPTION
**Pull request by andresmechali**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [CU-33a8fw3 update events to match the new types for pablo events](https://app.clickup.com/t/33a8fw3)

File: subsquid/schema.graphql [hunk 1]
```diff
@@ -167,20 +184,22 @@ type HistoricalVolume @entity {
   assetId: String!
 }
 
-type PabloTransaction @entity {
+type PabloSwap {
```
Comment (vimukthi-git) [inline subsquid/schema.graphql:187]: Track this for debugging and future proofing analytics.

**Review by vimukthi-git — APPROVED** [review]
LGTM, need to track the swaps and this should be good

**Review by PoisonPhang — APPROVED** [review]

File: subsquid/.prettierrc [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {
   "semi": true,
```
Comment (andresmechali) [inline subsquid/.prettierrc:2]: Placeholder comment to keep PR from merging

**Review by easteregg — APPROVED** [review]

**Review by easteregg — APPROVED** [review]

**Comment by andresmechali** [comment]
@mergify refresh

**Review by vimukthi-git — APPROVED** [review]
LGTM

**Pull request merged**